automatic/freecad/update_helper.ps1

```diff
@@ -46,6 +46,11 @@ function Get-FreeCad {
     [version]$version = ( ( ($version.Major), ($version.Minor), ($filler), ($version.Build) ) -join "." )
   }
   $vert = @{$true = "$version-$kind"; $false = "$version" }[( $kind -eq 'dev' )]
+# The url64 should always be valid
+$test = (Update-OnETagChanged -execUrl "${PreUrl}${url64}" -saveFile "${PSScriptRoot}\freecad.info" -kind $kind -OnETagChanged { @{} } -OnUpdated { @{} } -boule )
+if (($test -eq $true) -and ($kind -ne 'dev')) {
+	$vert = (Get-FixVersion $vert)
```

Comment:
Please follow the rules regarding the style specified in the `.editorconfig` file.

We do not use tab indentation, but instead 2 spaces.